commit/plugin_e2e_test.go

```diff
@@ -9,15 +9,18 @@ import (
 	"time"
```

Comment:
Changed current tests to use multiple outcomes (specifically `merkleroot.Outcome`). No new tests added. Maybe later we'll refactor this to have some mocks and make sure it's aggregating correctly without testing the underlying processors as already each processor will have their own unit tests.